packages/react-graphql-universal-provider/package.json

```
@@ -24,7 +24,7 @@
     "node": "^14.17.0 || >=16.0.0"
   },
   "dependencies": {
-    "@shopify/react-graphql": "^8.0.4",
+    "@shopify/react-graphql": "^9.0.0",
```

Lets widen the range to support both old and new versions of `@shopify/react-graphql`, if we don't want `@shopify/react-graphql-universal-provider` to be a major bump too

```suggestion
    "@shopify/react-graphql": "^8.0.4 || ^9.0.0",
```